tests/test_main_window.py

```diff
@@ -45,6 +45,15 @@ def main_window(qtbot, request):
     win = MainWindow(viewer=viewer, remote=request.param == "remote")
     win._mmc.loadSystemConfiguration("demo")
 
+    win.mda.SEQUENCE_META = {
```

Comment:
hmm, this feels like we're just making the test work with the new code. but not necessarily considering whether the test was actually "correct" already.

I need to take a closer look, but my immediate feeling here is that we've taken a method that takes a clean object (a useq MDAevent) and has a predictable response, and turned it into something that will only work correctly if some specific state has already been satisfied on the object (that's what this test "fix" is doing... it's making sure that the test has the state it needs pass). I'd like to take another look at why the mda_frame method now _also_ needs to know something about the SEQUENCE_META object.  will do so soon

Reply:
Sure! Maybe instead of:
```
def _on_mda_frame(self, image: np.ndarray, event: useq.MDAEvent):
        seq = event.sequence
        meta = self.mda.SEQUENCE_META.get(seq, {})

        if meta.get("mode") == "mda":
            ...
```
we can change it with:

           `if not "mode" in meta`

or something similar...not sure...